lms/djangoapps/grades/models.py

```diff
@@ -376,6 +423,14 @@ def _prepare_params_visible_blocks_id(cls, params):
         params['visible_blocks_id'] = params['visible_blocks'].hash_value
         del params['visible_blocks']
 
+    def remove_attempts(self):
```

Comment:
If this is a test-only method, we should name it as such.  Otherwise, it's confusing to see it available as a possible use case.